app.js

```diff
@@ -175,9 +175,11 @@ function renderResults(){
     ul.appendChild(li);
   }
   renderChart();
+  saveSettings();
 
   return ul;
 }
+getItem();
```

Comment:
Try to keep all your functional code towards the bottom of the page, it will save your team a lot of time in project week!